kernel/kernel-api/src/main/java/io/delta/kernel/PostCommitAction.java

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright (2025) The Delta Lake Project Authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.delta.kernel;
+
+import java.io.IOException;
+
+public interface PostCommitAction {
+
+  // Invokes the post commit action, implementation of action should be thread safe.
```

Comment:
Please use `/** */` for java method docs

Reply:
/** Invokes the post commit operation whose implementation must be thread safe. */